src/api/Jalasoft.TeamUp.Projects.API/Controllers/GetProjectById.cs

```diff
@@ -13,9 +13,9 @@
 
     public class GetProjectById
     {
-        private readonly IProjectsService projectsService;
+        private readonly IProjectService projectsService;
```

Comment:
```suggestion
        private readonly IProjectsService projectsService;
```
please keep the names in plural